src/explore-education-statistics-admin/src/pages/release/data/components/DraftApiDataSetsTable.tsx

```diff
@@ -46,88 +50,107 @@ export default function DraftApiDataSetsTable({
+                    {`v${draftVersion.version}`}
                   </Tag>
                 </td>
-              ) : null}
-              <td>{dataSet.title}</td>
-              <td>
-                <Tag
-                  colour={getDataSetVersionStatusTagColour(draftVersion.status)}
-                >
-                  {getDataSetVersionStatusText(draftVersion.status)}
-                </Tag>
-              </td>
-              <td>
-                <ButtonGroup
-                  className="govuk-!-margin-bottom-0"
-                  horizontalSpacing="m"
-                >
-                  <Link
-                    to={generatePath<ReleaseDataSetRouteParams>(
-                      releaseApiDataSetDetailsRoute.path,
-                      {
-                        publicationId,
-                        releaseId,
-                        dataSetId: dataSet.id,
-                      },
+                {hasLiveDataSets ? (
+                  <td>
+                    <Tag colour={latestLiveVersion ? 'blue' : 'grey'}>
+                      {latestLiveVersion?.version
+                        ? `v${latestLiveVersion?.version}`
+                        : 'N/A'}
+                    </Tag>
+                  </td>
+                ) : null}
+                <td>{dataSet.title}</td>
+                <td>
+                  <Tag
+                    colour={getDataSetVersionStatusTagColour(
+                      draftVersion.status,
                     )}
                   >
-                    {draftVersion.version === '1.0'
-                      ? 'View details'
-                      : 'View details / edit draft'}
-                    <VisuallyHidden>for {dataSet.title}</VisuallyHidden>
-                  </Link>
-                  {draftVersion.status !== 'Processing' && (
-                    <DeleteDraftVersionButton
-                      dataSet={dataSet}
-                      dataSetVersion={draftVersion}
+                    {getDataSetVersionStatusText(draftVersion.status)}
+                  </Tag>
+                </td>
+                <td>
+                  <ButtonGroup
+                    className="govuk-!-margin-bottom-0"
+                    horizontalSpacing="m"
+                  >
+                    <Link
+                      to={generatePath<ReleaseDataSetRouteParams>(
+                        releaseApiDataSetDetailsRoute.path,
+                        {
+                          publicationId,
+                          releaseId,
+                          dataSetId: dataSet.id,
+                        },
+                      )}
                     >
-                      Delete draft
-                      <VisuallyHidden> for {dataSet.title}</VisuallyHidden>
-                    </DeleteDraftVersionButton>
-                  )}
-                </ButtonGroup>
-              </td>
-            </tr>
-          ),
+                      {draftVersion.version === '1.0' || !canUpdateRelease
```

Comment:
Is it not possible to edit a draft v1.0 version then?

Reply:
Not currently, there's nothing to edit as you can't change the title or summary and there aren't any mappings for the first version.

Reply:
Cool, makes sense thanks.